packages/set/src/ValueObject/SetList.php

```diff
@@ -211,6 +211,11 @@ final class SetList
      */
     public const KDYBY_TRANSLATOR_TO_CONTRIBUTTE_TRANSLATION = __DIR__ . '/../../../../config/set/kdyby-translator-to-contributte-translation.php';
 
+    /**
+     * @var string
+     */
+    public const LAMINAS_SERVICEMANGER_40 = __DIR__ . '/../../../../config/set/laminas-servicemanager-40.php';
```

Comment:
How is laminas project versioned?

If it's a monorepo, we can have just `LAMINAS_40` here

Reply:
The laminas project is not a monorepo, per-component has its own live so upgrade path may need per-component basis.

Reply:
I see.

Let me know when it's ready to merge